### PR TITLE
day 6 - WIP

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Note this is a WIP, what do you think ? @backbee/core-team

### CODE REVIEW

**Comment by eric-chau** [comment]
Seems to be a good first step =)

File: doc/day6.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+Each bundle need a configuration file, that contains configuration of Bundle loaded by BackBee. Look at this configuration reference file:
+
+```yaml
+bundle:
+    name: Foo bundle
+    description: A description of the bundle, displayed in extensions menu
+    author: John Doe <john.doe@backbee.com>
+    version: 1
+    enable: true
+    bundle_loader_recipes:
```
Comment (ndufreche) [inline doc/day6.md]: This part is optional, I think is better to not put it.

File: doc/day6.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    </div>
+    <div {{ this.bbcontent(author, {'class': 'quote-author'})|raw }}>
+        {{ this.render(author)|raw }}
+    </div>
+</div>
+```
+
+That's simple.
+
+Additionaly, you can provide a logo for this new Class Content: the only thing to do is to create an image called *Quote.png* in `/bundle/QuoteBundle/Resources/img/contents/Quote/`.
```
Comment (ndufreche) [inline doc/day6.md]: We can provide the picture.

File: doc/day6.md (hunk 1 above)
Comment (mickaelandrieu) [inline doc/day6.md]: agree, I remove it.

File: doc/day6.md (hunk 2 above)
Comment (mickaelandrieu) [inline doc/day6.md]: yeah, I think the old "blogbee" website need an update :/ ping @clementbonfils

**Comment by ndufreche** [comment]
Good job @mickaelandrieu :+1:

**Comment by mickaelandrieu** [comment]
thank you for review @ndufreche.

**Pull request merged**